apcd-cms/src/apps/admin_regis_table/templates/create_submitter_success.html

```diff
@@ -0,0 +1,13 @@
+{% extends "standard.html" %}
+{% block content %}
+<div class="container">
+    {% include "nav_cms_breadcrumbs.html" %}
+
+    <h1>Create Submitter</h1>
+    <hr />
+    <p>
+        Creation of this submitter was successful.
+    </p>
+    <a class="c-button c-button--primary" href="/administration/list-registration-requests">Back to List Registrations</a>
+</div>
+{% endblock %}
```

Comment:
**Form Submission Success Eludes Me.** I seem unable to ever load this page. How do I submit the form successfully? What format do the fields expect?

Reply:
I got answers.

1. **I don't.** Form can not be submitted successfully locally, because of no test database access (real or otherwise).
2. From database: payor_code, integer; submitter_code, 8 char; encryption_key, 30 char.

I added client-side validation (4037c48).